internal/lvmd/command/lvm_lv_attr_test.go

```diff
@@ -0,0 +1,219 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func TestParsedLvAttr(t *testing.T) {
+	NoError := func(t *testing.T, err error, args ...string) bool {
```

Comment:
I think you do not need to make the local variable public.
```suggestion
	noError := func(t *testing.T, err error, args ...string) bool {
```